frontend/public/components/monitoring.tsx

```diff
@@ -986,13 +981,19 @@ const EditSilence = connect(silenceParamToProps)(({loaded, loadError, silence})
   </StatusBox>;
 });
 
-const CreateSilence = () => {
+export const CreateSilence = () => {
   const matchers = _.map(getURLSearchParams(), (value, name) => ({name, value, isRegex: false}));
   return _.isEmpty(matchers)
     ? <SilenceForm saveButtonText="Create" title="Create Silence" />
     : <SilenceForm defaults={{matchers}} saveButtonText="Create" title="Silence Alert" />;
 };
 
+export const AlertManagerYAML = () => {
```

Comment:
Will need to display an error if the `alertmanager-main` secret is not found.

Reply:
I do display an error [here](https://github.com/dtaylor113/console/blob/alerting/frontend/public/components/secrets/create-secret.tsx#L890)